src/main/webapp/view/view.js

```diff
@@ -53,3 +54,9 @@ function getLectureId() {
   const urlParams = new URLSearchParams(window.location.search);
   return urlParams.get(PARAM_ID);
 }
+
+/** Sets the lecture name in `header-text`. */
+function setLectureName() {
+  const headerText = document.getElementById('header-text');
+  headerText.innerText = window.LECTURE.lectureName;
```

Comment:
Can you remove the `href` attribute so clicking doesn't take us to the About page?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Element/removeAttribute

Reply:
It is just a `<h2>` right now

Reply:
Ah yes, approved.